src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1212,4 +1212,13 @@
   <data name="DoNotUseCreateEncryptorWithNonDefaultIVDescription" xml:space="preserve">
     <value>Symmetric encryption should always use a non-repeatable initialization vector to prevent dictionary attacks.</value>
   </data>
+  <data name="DoNotCreateTaskCompletionSourceWithWrongArgumentsDescription" xml:space="preserve">
+    <value>Argument '{0}' needs to be of type TaskCreationOptions enum, not of TaskContinuationsOptions enum.</value>
+  </data>
+  <data name="DoNotCreateTaskCompletionSourceWithWrongArgumentsMessage" xml:space="preserve">
+    <value>Argument contains TaskContinuationsOptions enum instead of TaskCreationOptions enum.</value>
+  </data>
+  <data name="DoNotCreateTaskCompletionSourceWithWrongArgumentsTitle" xml:space="preserve">
+    <value>Argument contains TaskContinuationsOptions enum instead of TaskCreationOptions enum.</value>
```

Comment:
```suggestion
    <value>Argument passed to TaskCompletionSource constructor must be TaskContinuationsOptions enum instead of TaskCreationOptions enum.</value>
```